src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=" "
+      method="post"
+      data-qa="big"
+      class="search-bar big_search-bar"
+    >
+      <label class="search-bar__label search-bar__label_big">
+        <input
+          class="search-bar__input search-bar__input-big"
+          name="input-big"
```

Comment:
Should be camelCase, read in the checklist
(https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#attributes)